lib/generators/clearance/install/templates/db/migrate/create_users.rb

```diff
@@ -1,4 +1,4 @@
-class CreateUsers < ActiveRecord::Migration
+class CreateUsers < ActiveRecord::Migration<%= migration_version %>
```

Comment:
I assume this is because despite the `.rb` extension, `.erb` is actually supported here. I'm fine with this.